app/src/main/res/layout/activity_event_details.xml

```diff
@@ -12,7 +12,7 @@
         android:paddingLeft="@dimen/activity_horizontal_margin"
         android:paddingRight="@dimen/activity_horizontal_margin"
         android:paddingTop="@dimen/activity_vertical_margin"
-        tools:context=".event.dashboard.EventDetailsActivity">
```

Comment:
Dashboard means a home screen where all controls are present, whereas this is just a detail view of previous list of events activity